merkle/memory_merkle_tree_test.go

```diff
@@ -391,8 +435,11 @@ func TestReferenceMerklePathSanity(t *testing.T) {
 	}
 
 	for _, path := range testPaths {
-		referencePath := referenceMerklePath(data[:path.snapshot], path.leaf,
+		referencePath, err := referenceMerklePath(data[:path.snapshot], path.leaf,
 			mt.hasher)
```

Comment:
nit/pre-existing: move params to line above.

Reply:
done